rust/cardano-chain-follower/src/turbo_downloader/mod.rs

```diff
@@ -426,11 +434,24 @@ impl ParallelDownloadProcessor {
         self.download()
     }
 
+    /// Call the work queue receiver.
+    /// This is a helper function to pause and resume the stats thread.
+    fn call_work_queue_receiver(
+        chain: Network, worker_id: usize, work_queue: &Receiver<usize>,
+    ) -> Result<usize, RecvError> {
+        let thread_name = format!("{}::{worker_id}", stats::thread::name::PARALLEL_DL_WORKER);
```

Comment:
Move the thread name creation to the thread itself, and pass as a parameter (&str)